source/administration/changelog.md

```diff
@@ -80,6 +80,8 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
 ## Release v5.30 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
+- **v5.30.2, released 2021-01-16**
+  - Fixed an issue where a column in the ``ThreadMemberships`` table was missing. Admins planning to enable [Collapsed Reply Threads](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.30.2 or later.
```

Comment:
Fixed an issue where the ``UnreadMentions`` column was missing in the ``ThreadMemberships`` for servers upgrading from an earlier version. Admins planning to enable [Collapsed Reply Threads](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.30.2 or later.